### ISSUE TITLE
No more Profile?

### ISSUE THREAD

**Issue by aburrell** [timduly4/pyglow#111]
Profile functionality was removed, was this intentional?

**Comment by bharding512** [comment]
I don't think this pull request was ever accepted: https://github.com/timduly4/pyglow/pull/25

**Comment by timduly4** [comment]
It is almost trivial to put this back in (e.g.):

```
class Profile:
    def __init__(self, dn, lat, lon, start_height, end_height, spacing=1):
        altitudes = range(start_height, end_height, spacing)
        self.points = [Point(dn, lat, lon, altitude) for altitude in altitudes)
```

**Comment by aburrell** [comment]
It would be nice to have (back) in, since it's how I've continued to use pyglow.

In light of #25 never being accepted, a pull request that stale should probably be closed.

**Comment by timduly4** [comment]
We'll keep this as an outstanding issue, then.  For better or worse, I'm behind on several pyglow issues at the moment-- these are the perils of an open-source project...
